.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+<!--- 
+Replace this comment with a description of the problem.
+
+Please help us by providing all of the details listed below. A complete and thoughtful issue request with research on possible root causes and diagnostics goes a very long way in helping us rapidly get your issues resolved. 
+--> 
+
+**Reproduction Steps:**
+
+**OS and version:**    
+
```

Comment:
I think that some important info that we need here are:
- rhche version
- openshift and k8s versions